profiles/common/modules/features/multisite_notifications/multisite_notifications_core/multisite_notifications_core.module

```diff
@@ -340,14 +126,14 @@ function _multisite_notifications_core_delete_anonymous_account($mail, $waiting_
   // Block user until user confirms.
```

Comment:
Can't this whole method be removed? It's related to the anonymous feature we want to get rid off AFAIK.